cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -84,7 +84,9 @@ func (q *ChannelMessageQueue) addListMessageToQueue(nodeID string, msg *beehiveM
 
 	messageKey, _ := getListMsgKey(msg)
 
-	nodeListStore.Add(msg)
+	if err := nodeListStore.Add(msg); err != nil {
+		klog.Errorf("failed to add msg: %s", err)
```

Comment:
klog.Errorf("Failed to add msg, err: %v", err)

Reply:
Should return here.

Reply:
Error strings should not  be capitalized, see https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Add return.